### DISCUSSION TITLE
iOS doesnt reopen after force close.

### DISCUSSION THREAD

**Discussion by jeo1kim** [gdelataillade/alarm#91] [Q&A]
I force close the app and reopen and it just shuts down.. is this happening for everyone?

**Comment by gdelataillade** [comment]
Hey @jeo1kim

Thanks for sharing your issue.

For every iOS app, you need to build the app in release mode to be able to kill it and reopen it.

Try with:
`flutter run --release`

Let me know if it solves your issue.

**Reply by jeo1kim** [reply]
Thank you! that solved it! :D
